code/modules/mob/living/carbon/carbon.dm

```diff
@@ -1080,6 +1080,16 @@ GLOBAL_LIST_INIT(ventcrawl_machinery, list(/obj/machinery/atmospherics/unary/ven
 /mob/living/carbon/proc/eat(obj/item/reagent_containers/food/toEat, mob/user, bitesize_override)
 	if(!istype(toEat))
 		return 0
+	if(iscarbon(user))
```

Comment:
this check is unnecessary, this proc should never be called on anything other than a carbon mob